pkg/collector/corechecks/containers/generic/filters.go

```diff
@@ -51,6 +51,13 @@ func (f LegacyContainerFilter) IsExcluded(container *workloadmeta.Container) boo
 		return false
 	}
 
+	if container.Owner != nil && container.Owner.Kind == workloadmeta.KindKubernetesPod {
```

Comment:
can we create a func to wrap this condition, because we use it in several places

something like: 
```suggestion
	if isPodExcluded(container.Owner, workloadmeta.KindKubernetesPod) {
```